manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -301,6 +305,11 @@ jobs:
         agent:
           services:
             gorouter: {}
+      racoon:
+        verify_certificate: "on"
```

Comment:
Seems like this would live better in the global `properties.racoon` alongside `certificate_authority_cert`. Could you move it and remove it from the cell job?